src/app/components/shared/baw-client/baw-client.component.ts

```diff
@@ -66,13 +71,15 @@ export class BawClientComponent extends withUnsubscribe() implements OnInit {
     }
 
     // Update iframe size every 250 milliseconds
-    interval(250)
+    interval(this.iframeUpdateIntervalMs)
```

Comment:
I noticed a bug in this with the version before this patch: if the iframe does not load the size just keeps increasing indefinitely.

Can you reproduce?

Reply:
I did know about the bug, but fixing it was kinda tricky and it only occurs if the baw client fails to load. The main issue is that if the client fails to load, i have no way of knowing. I also have no way of knowing if the previous size increase was because the iframe grew, or because the last iframe interval caused the change, Also, the amount the iframe grows by each step is browser dependent (50px in firefox, 57 for everyone else i can try).

Reply:
I could make a solution, but there's not much i can think of which doesn't add a potential edge case which affects the successful deploy. Especially as this only occurs on a fault deploy, and should be fairly easy to identify.

Reply:
yeah, fine to ignore